Blaise-Shyaka/Politico
# [#171720455] Create political party API endpoint

# What does this PR do?

 It creates an API endpoint allowing the admin to create a political party

# Description of the tasks to be completed

- Write tests for the `POST /parties` endpoint
- Write a controller for that endpoint and mount it on a router

# How should this be manually tested?

- Clone this repository
- Run `npm install` to install all dependencies and `npm run test` to run tests
- 11 test cases should pass

# What are the relevant pivotal tracker stories?

#171720455
